src/screens/NoticeboardIndexScreen.tsx

```diff
@@ -0,0 +1,99 @@
+import { StackScreenProps } from '@react-navigation/stack';
+import React, { useCallback, useContext, useState } from 'react';
+import { useQuery } from 'react-apollo';
+import { ActivityIndicator, RefreshControl, SectionList } from 'react-native';
+
+import { EmptyMessage, LoadingContainer, SafeAreaViewFlex, SectionHeader } from '../components';
+import { colors, texts } from '../config';
+import { graphqlFetchPolicy, parseListItemsFromQuery } from '../helpers';
+import { useStaticContent } from '../hooks';
+import { useRenderItem } from '../hooks/listHooks';
+import { NetworkContext } from '../NetworkProvider';
+import { getQuery, QUERY_TYPES } from '../queries';
+
+import { ListHeaderComponent } from './NestedInfoScreen';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+/* eslint-disable complexity */
+export const NoticeboardIndexScreen = ({ navigation, route }: StackScreenProps<any>) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp });
+  const [refreshing, setRefreshing] = useState(false);
+
+  const consentForDataProcessingText = route.params?.consentForDataProcessingText ?? '';
+  const content = route.params?.content ?? '';
+  const query = route.params?.query ?? '';
+  const queryVariables = route.params?.queryVariables ?? {};
+  const subQuery = route.params?.subQuery ?? '';
+  const title = route.params?.title ?? '';
+
+  const { data, loading, refetch } = useQuery(getQuery(query), {
+    fetchPolicy,
+    variables: queryVariables
+  });
+  const listItems = parseListItemsFromQuery(query, data, consentForDataProcessingText, {
+    queryVariables
+  });
+
+  const {
+    data: dataHtml,
+    loading: loadingHtml,
+    refetch: refetchHtml
+  } = useStaticContent<string>({
+    name: content,
+    type: 'html',
+    skip: !content
+  });
+
+  const onRefresh = useCallback(async () => {
+    setRefreshing(true);
+    if (isConnected) {
+      await refetch?.();
+      await refetchHtml?.();
+    }
+    setRefreshing(false);
+  }, [isConnected, refetch]);
+
+  const renderItem = useRenderItem(QUERY_TYPES.PUBLIC_JSON_FILE, navigation);
+
+  if (loading && !data)
+    return (
+      <LoadingContainer>
+        <ActivityIndicator color={colors.accent} />
+      </LoadingContainer>
+    );
+
+  if (!data) {
+    return <EmptyMessage title={texts.noticeboard.emptyTitle} />;
+  }
+
+  const sectionData = [{ title: title, data: listItems }];
+
+  return (
+    <SafeAreaViewFlex>
+      <SectionList
```

Comment:
why is this a section list? what are the sections? there are no section or am i wrong? we have a list with three entries but no sections?

Reply:
I can remove `SectionList`. Yes, there is no section, but I don't like the look when I don't use `SectionList`.

Reply:
yes please, we do not have sections here

Reply:
I think everything's all right now.

https://github.com/ikuseiGmbH/smart-village-app-app/pull/698/commits/00a19f8f9ba995dcfbf34843f7404fcef1a7816f

![Simulator Screen Shot - iPhone 8 Plus - 2022-11-21 at 12 44 13](https://user-images.githubusercontent.com/11755668/203043333-11ab45fb-850c-415e-9ede-08bd34d70393.png)